core/src/main/java/com/graphhopper/routing/lm/LandmarkStorage.java

```diff
@@ -464,10 +464,10 @@ protected IntHashSet findBorderEdgeIds(SpatialRuleLookup ruleLookup) {
         IntHashSet inaccessible = new IntHashSet();
         while (allEdgesIterator.next()) {
             int adjNode = allEdgesIterator.getAdjNode();
-            SpatialRule ruleAdj = ruleLookup.lookupRule(nodeAccess.getLatitude(adjNode), nodeAccess.getLongitude(adjNode));
+            SpatialRule ruleAdj = ruleLookup.lookupRules(nodeAccess.getLatitude(adjNode), nodeAccess.getLongitude(adjNode)).getRules().get(0);
 
             int baseNode = allEdgesIterator.getBaseNode();
-            SpatialRule ruleBase = ruleLookup.lookupRule(nodeAccess.getLatitude(baseNode), nodeAccess.getLongitude(baseNode));
```

Comment:
@otbutz if the first SpatialRule is always available should we provide a convenient method for this? Or will it be rather rare use case?

Reply:
No, it is not always available per se. The landmark use case is a bit special in that respect.

Reply:
Ok, so if no rule is available the list could be empty (unlike before where you got SpatialRule.EMPTY?)

Reply:
Correct. If no rules apply, an empty List is returned.

Reply:
There seems to be a problem with this code: https://github.com/graphhopper/graphhopper/issues/2020